### ISSUE TITLE
output text files

### ISSUE THREAD

**Issue by macmanes** [mdshw5/fastqp#27]
I see that the info is output in an unstructured way on the stout, but for those of us that would like to do some other types of viz, it would be nice to have the raw data. 

Thanks, Matt

**Comment by mdshw5** [comment]
Absolutely. I was thinking I would have a JSON output. Would this be acceptable?

**Comment by macmanes** [comment]
I _think_. Assuming there is a good json parser in R.

**Comment by mdshw5** [comment]
jsonlite is a very solid package: https://cran.r-project.org/web/packages/jsonlite/index.html

The reason I ask is that the output is somewhat structured already as:

| sample id     | observation | cycle         | value            |
|-------------|-------------|-----------|-------------|

Which you should be able to parse in R using `tidy::spread`.
